frontend/src/components/molecules/ExpandableQuote/index.tsx

```diff
@@ -1,22 +1,37 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import './style.css';
 import AlertIcon from '../../atoms/icons/alert';
 import { ArrowDownIcon, ArrowUpIcon } from '../../atoms/icons/fr-fi-icons';
 
-type Props = { title: string; large?: boolean; openOnMount?: boolean };
+type Props = {
+  title: string;
+  large?: boolean;
+  openOnMount?: boolean;
+  scrollOnMount?: boolean;
+};
 
 export const ExpandableQuote: React.FC<Props> = ({
   title,
   large = false,
   openOnMount = false,
+  scrollOnMount = false,
   children,
 }) => {
   const [expanded, setExpanded] = useState(false);
+  const ref = useRef(null);
+
+  useEffect(() => {
+    if (scrollOnMount) {
+      // @ts-ignore
+      ref.current.scrollIntoView();
```

Comment:
No need for  ts-ignore if the type is set l.21. 
`ref.current?.scrollIntoView()` to be safe